packages/excalidraw/fractionalIndex.ts

```diff
@@ -0,0 +1,266 @@
+import { mutateElement } from "./element/mutateElement";
+import { ExcalidrawElement } from "./element/types";
+import {
+  generateKeyBetween as _generateKeyBetween,
+  generateJitteredKeyBetween as _generateJitteredKeyBetween,
+  generateNKeysBetween as _generateNKeysBetween,
+  generateNJitteredKeysBetween as _generateNJitteredKeysBetween,
+  base62CharSet as _base62CharSet,
+  indexCharacterSet,
+} from "fractional-indexing-jittered";
+import { ENV } from "./constants";
+import { Mutable } from "./utility-types";
+
+type FractionalIndex = ExcalidrawElement["fractionalIndex"];
+
+const base36CharSet = indexCharacterSet({
+  chars: "0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ",
+  firstPositive: "A",
+  mostPositive: "Z",
+});
+
+const { chars, firstPositive, mostPositive, mostNegative } = _base62CharSet();
+
+const base62CharSet = indexCharacterSet({
+  chars,
+  firstPositive,
+  mostPositive,
+  mostNegative,
+  jitterRange: 6,
```

Comment:
This was set to `6` supposedly to increase jitter range, but it works differently. The default range for base62 is `47665`

Reply:
Removed the jitter range, as it works differently.

<img width="900" alt="Screenshot 2024-01-02 at 11 22 13" src="https://github.com/excalidraw/excalidraw/assets/20387666/9b331bc4-2d3f-4eec-9ce0-053e3532e747">